src/design/components/organisms/MetadataContainer/molecules/MetadataContainerRow.tsx

```diff
@@ -177,13 +177,13 @@ const Container = styled.div`
     }
 
     &:disabled {
-      color: ${({ theme }) => theme.colors.text.subtle};
+      color: ${({ theme }) => theme.colors.text.primary};
+      opacity: 1;
```

Comment:
as a slight feedback, I personally find it confusing.
Disabled buttons will have no style change in comparison to others so people who don't directly go to hover it, or use mouse and keyboard to navigate will never know what is disabled or not at first glance.

Reply:
Let's wait for user feedback